about.html

```diff
@@ -0,0 +1,153 @@
+          <li class="info"><a href="#contact">news</a></li>
+          <li class="info"><button type="button" class="nav-button">cc company</button></li>
+        </ul>
+      </div>
+    </nav>
+
+    <div class="burger" id="burger">
+      <div class="line"></div>
+      <div class="line"></div>
+      <div class="line"></div>
+    </div>
+    <!--Section Hero-->
+    
+      <section class="introduction" id="home">
+
+        <h1 class="about-second-heading">THE BIGGEST<br /> FASHION EVENT 2022</h1>
+
+        <div class="about-para-border">
+          <p>
+            The DC Fashion Show is a conference brings fashion to come together from all over the world in order to
+            share their crazy ideas and talk about trendigs, those discussions can be more like in our own small nation,
+            but also in terms of fashion.
+            <span class="highlight">fashion </span> world
+          </p>
+        </div>
+
+        <div class="about-contact">
+          <p>
+            Please contact us per Email for any further questions about DC Fashion
+            Show!<br /><strong><u>dcfashionshow@ccomunity.org</u></strong>
+          </p>
+        </div>
+      </section>
+    </header>
+    <section class="create-common">
+      <h1>Creative Designs Soft-Stiches 2022</h1>
+      <br />
+      <br />
+      <p>
+        The name Soft-Stiches came from the magic hands on a fabric. A stitch
+        with the touch of magic..
+      </p>
+      <br />
+      <br />
+
+      <img class="card-logo" src="./images/text-1653551772639.jpg" alt="Mirela Pitt's picture">
+    </section>
+
+
+    <section class="past-reviews">
+      <h1>See the past Soft-Stiches Global Conference</h1>
+      <hr>
+      <p>
+        The two previous Soft-Stiches Global conference held in France 2016
+        and Tokoyo 2019
+      </p>
+
+      <div class="card-preview">
+        <div class="card-one">
+          <div class="cardBg"></div>
+          <h1 class="index">2016</h1>
+          <p>Soft-Stiches in France 2016</p>
+        </div>
+        <div class="card-two">
+          <div class="cardBg"></div>
+          <h1 class="index">2019</h1>
+          <p>Soft-Stiches in Tokoyo 2019</p>
+        </div>
+      </div>
+    </section>
+    </main>
+    <!-- Footer -->
+
+    <section>
+      <div class="foot foot-about">
+        <div class="foot-heading">
+          <h1>Partner</h1>
+          <hr>
+        </div>
+        <div class="partners">
+          <ul>
+          <li id="Stitch-up">STITCH UP</li>
+          <li id="Air-ups">Hope Fashion</li>
+          <li id="Mozzi-ups">Hot Make-up</li>
+          <li id="Air-up">Air UP</li>
+          <li id="Mozzi-up">Mozzi UP</li>
+        </ul>
+        </div>
+      </div>
+    </section>
+
+      <footer class="close-foot-aboutcolor">
+        <div class="about-close-foot">
+          <div class="logo-about">
+            <h3>Soft-Stiches</h3>
+          </div>
```

Comment:
- [ ] Kindly as requested by the previous reviewer, add a logo to your footer section, just before the soft-stitches header like in the design given.